app/model/Name.scala

```diff
@@ -9,7 +9,7 @@ object Name {
 
   def fromOktaUser(oktaUser: OktaUser): Name =
     Name(
-      title = nonNullNonEmpty(oktaUser.getProfile.getTitle),
+      title = nonNullNonEmpty(oktaUser.getProfile.getHonorificPrefix),
```

Comment:
it looks like what we can title we store in the 'honorific prefix' in Okta rather than the 'title' field.
See:
https://github.com/guardian/identity/blob/185068817d5e1a0b603017a5438d91bdcd497da7/identity-api/src/main/scala/com/gu/identity/api/transforms/OktaResponseTransforms.scala#L41